apis/v1beta1/vspheremachine_types.go

```diff
@@ -68,6 +68,30 @@ type VSphereMachineSpec struct {
 	//
 	// +optional
 	GuestSoftPowerOffTimeout *metav1.Duration `json:"guestSoftPowerOffTimeout,omitempty"`
+
+	// NamingStrategy allows configuring the naming strategy used when calculating the name of the VSphereVM.
```

Comment:
calculating -> preparing
as it's not a math calculation.

> name of the VSphereVM.

this field is inside VSphereMachineSpec not  VSphereVMSpec